libraries/AP_Baro/examples/BARO_generic/BARO_generic.cpp

```diff
@@ -62,7 +87,9 @@ void loop()
                             (double)barometer.get_altitude(),
                             (double)barometer.get_climb_rate(),
                             (unsigned)read_time);
-    } else {
+    } 
+    //if stipulated time has not passed between two distinct readings, delay the program for a millisecond
+    else {
```

Comment:
don't break the code style for comments

Reply:
I have made the changes you asked for. Please check

Reply:
still here

Reply:
So you want else on a seperate line? that was how it was in the original PR

Reply:
no the else shouldn't be break , it should stay } else { and put the comment after or under